fml/platform/win/wstring_conversion.h

```diff
@@ -5,24 +5,15 @@
 #ifndef FLUTTER_FML_PLATFORM_WIN_WSTRING_CONVERSION_H_
 #define FLUTTER_FML_PLATFORM_WIN_WSTRING_CONVERSION_H_
 
-#include <codecvt>
-#include <locale>
 #include <string>
 
 namespace fml {
 
-using WideStringConvertor =
-    std::wstring_convert<std::codecvt_utf8_utf16<wchar_t>, wchar_t>;
+// Returns a UTF-8 encoded equivalent of a UTF-16 encoded input string.
```

Comment:
Nit: maybe mention wide string in these comments? They currently read identically to the comments for the non-wide versions which is a bit confusing.

Reply:
Done.